c/driver/postgresql/connection.cc

```diff
@@ -478,19 +479,35 @@ AdbcStatusCode PostgresConnection::GetInfo(struct AdbcConnection* connection,
   for (size_t i = 0; i < info_codes_length; i++) {
     switch (info_codes[i]) {
       case ADBC_INFO_VENDOR_NAME:
-        infos.push_back({info_codes[i], "PostgreSQL"});
+        if (RedshiftVersion()[0] > 0) {
+          infos.emplace_back(info_codes[i], "Redshift");
+        } else {
+          infos.push_back({info_codes[i], "PostgreSQL"});
```

Comment:
I should have been clearer but all of the push_back's here I think are better with emplace_back

Reply:
C++...what a language.

Well in this case either is likely fine. I am under the impression that emplace_back would avoid any calls to the copy constructor of the list element, along with any move constructors that need to be called when the vector is resized. In this particular case it probably doesn't make a difference; maybe something to just look at when performance is more critical